std/algorithm/comparison.d

```diff
@@ -546,6 +544,17 @@ elements.
 
 If the ranges are strings, $(D cmp) performs UTF decoding
 appropriately and compares the ranges one code point at a time.
+
+Params:
+    pred = The predicate used for comparison.
+    r1 = The first range.
+    r2 = The second range.
+
+Returns:
+    0 if both all ranges are entirely equal. -1 if $(D r1) is less than $(D r2)
```

Comment:
Also, "r1 is less than r2" may not be very obvious when r1 and r2 are ranges. What about "if the first differing element of r1 is less than the corresponding element of r2"? Ditto for the greater than case.